src/Components/Inputs/Input.scss

```diff
@@ -82,14 +82,15 @@
 .cf-input-icon {
   pointer-events: none;
   top: 50%;
-  position: absolute;
-  z-index: $cf-input--status-z;
-  transition: color 0.25s ease;
   font-size: 1.1em;
+  transform: translate(-50%, -50%);
 }
 
-.cf-input-icon {
-  transform: translate(-50%, -50%);
+.cf-input-icon,
+.cf-input-clear-btn {
```

Comment:
This might be a .cf-input__clear-btn based on BEM methodology but I could be wrong